config/core/resources/eventtype.yaml

```diff
@@ -22,6 +22,132 @@ metadata:
 spec:
   group: eventing.knative.dev
   versions:
+  - name: v1beta3
+    served: false
```

Comment:
The new version is not served as a precaution for now?

Reply:
Yeah, I figured we could serve it when we update the reconciler to reconcile the new type